packages/neos-ui-editors/src/Editors/Range/index.js

```diff
@@ -68,11 +68,11 @@ class RangeEditor extends PureComponent {
     };
 
     render() {
-        const {options} = this.props;
+        const options = Object.assign({}, this.constructor.defaultOptions, this.props.options);
```

Comment:
i think i dont understand `this.constructor.defaultOptions` but it seems having to do with: `static defaultProps options` right?

Reply:
Fixed it. Strange that it worked also like that

Reply:
i looks like the 'static defaultProps' is a react thing and works out of box: https://stackoverflow.com/questions/38786973/how-to-set-component-default-props-on-react-component

so maybe your good to go using only this.props.options and the rest is handeled backstage...